cobbler/tftpgen.py

```diff
@@ -683,15 +660,11 @@ def write_pxe_file(self, filename, system, profile, distro, arch: Archs, image=N
 
         if distro and distro.os_version.startswith("esxi") and filename is not None:
             append_line = "BOOTIF=%s" % (os.path.basename(filename))
-        elif "initrd_path" in metadata and arch not in [Archs.PPC, Archs.PPC64, Archs.ARM]:
+        elif "initrd_path" in metadata and not arch:
```

Comment:
This looks wrong.
I wonder why PPC PPC64 and ARM are not going to append an initrd.

But these seem to be VMware (esxi) specific conditions, not bootloader specific ones.
I'd just keep "... and arch not in [Archs.PPC, Archs.PPC64, Archs.ARM]:"

Reply:
There is no reason for arch-specific handling.

This is bootloader-specific. yaboot (and lilo I think) append the kernel commandline differently compared to otehr bootloaders.

Reply:
removed